hack/lib/version.sh

```diff
@@ -135,40 +144,40 @@ kube::version::load_version_vars() {
   source "${version_file}"
 }
 
-kube::version::ldflag() {
```

Comment:
because this generates 3 entries for an array and is called repeatedly for the same array it is difficult to correctly wrap it without splitting concerns etc.

instead i've nested it in the only caller (`kube::version::ldflags`) and have it append to the array directly. much simpler.